_includes/v22.2/known-limitations/copy-syntax.md

```diff
@@ -4,12 +4,12 @@ CockroachDB does not yet support the following `COPY` syntax:
 
   - [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/85571)
 
-- Various unsupported `COPY` options (`FORMAT`, `FREEZE`, `QUOTE`, etc.)
+- Various `COPY` options (`FORMAT`, `FREEZE`, `QUOTE`, etc).
```

Comment:
I think this should be:

> (`FORMAT`, `FREEZE`, `QUOTE`, etc.).

The period outside the parenthesis for the whole sentence, but the period to end "etc" since it's an abbreviation.